TGC/decisions/NET.txt

```diff
@@ -355,6 +355,9 @@ political_decisions = {
 			badboy = 5
 			set_country_flag = artois_claimed
 			FRA_404 = { add_core = NET }
+			405 = { add_core = NET }
+			400 = { add_core = NET }
+			399 = { add_core = NET }
```

Comment:
might be cleaner to do this by scoping the respective states of these provinces and adding a Dutch core to all the French cores in those regions.